src/app/products/loan-products/loan-product-stepper/loan-product-organization-unit-step/loan-product-organization-unit-step.component.ts

```diff
@@ -33,6 +33,9 @@ export class LoanProductOrganizationUnitStepComponent implements OnInit {
   // loan terms and condition templates
   enableTermsAndConditions: boolean = false;
   loanProductTemplates: any = [];
+
+  isQualificationRequired: boolean = false;
```

Comment:
Btw question.
Remember that we noticed that if qualification is required the the 'Qualification Window Period' should be the same as the 'Repayment Start Period'. Is it catered for in this. If not maybe we can use this PR to look into it?
I'll approve the PR though, just in case

Reply:
@Vikeln I created a different ticket for it, but it will not be able to be handled in the next sprint though.